### PR TITLE
[DEV] Adding Audit log for the Activity and Task form

### PR DESCRIPTION
**Pull request by jananjoy**
Tâche #231 Adding Audit log for the Activity and Task form 
- Changes the project task model and views for  Audit log in Activity and Task Form
- Remove overriding of the  write method
- Do the language translation for the Project event module [related with Audit log functionalities]

### CODE REVIEW

File: project_event/data/auditlog_rule.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="auditlog_project_task_rule" model="auditlog.rule">
+        <field name="name">project Task Logs</field>
```
Comment (rimbendhaou) [inline project_event/data/auditlog_rule.xml]: Project Task Logs

File: project_event/i18n/fr.po [hunk 2]
```diff
@@ -100,6 +108,12 @@ msgstr "Wizard d'activités"
 msgid "Activity wizard"
 msgstr "Wizard d'activités"
 
+#. module: project_event
+#: model:ir.actions.act_window,name:project_event.action_activity_to_auditlog_log
+#: model:ir.model.fields,field_description:project_event.field_project_task_activty_log_count
+msgid "Activty Logs"
```
Comment (rimbendhaou) [inline project_event/i18n/fr.po]: Activity Logs

File: project_event/models/project_task.py [hunk 3]
```diff
@@ -105,6 +105,30 @@ class Task(models.Model):
     reservation_event_id = fields.Integer(
         string='Reservation event',
     )
+    activty_log_count = fields.Integer(
```
Comment (rimbendhaou) [inline project_event/models/project_task.py]: @jananjoy we need only one field to count logs

File: project_event/models/project_task.py [hunk 4]
```diff
@@ -105,6 +105,30 @@ class Task(models.Model):
...
+        string='Activty Logs',
+        compute='_compute_activty_log_count',
+    )
+    task_log_count = fields.Integer(
+        string='Task Logs',
+        compute='_compute_task_log_count',
+    )
+
+    def _compute_activty_log_count(self):
```
Comment (rimbendhaou) [inline project_event/models/project_task.py]: @jananjoy why you define same method twice?

File: project_event/views/auditlog_log.xml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record model="ir.actions.act_window" id="action_activity_to_auditlog_log">
```
Comment (rimbendhaou) [inline project_event/views/auditlog_log.xml]: @jananjoy we can use only one act_window for task logs and activity logs

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/i18n/fr.po [hunk 6]
```diff
@@ -724,3 +740,6 @@ msgstr "Wizard de création d'événement à partir d'un modèle"
 #: model:ir.model,name:project_event.model_project_task_wizard
 msgid "Wizard Event Task Creation from template"
 msgstr "Wizard de création de tâche d'événement à partir d'un modèle"
+
```
Comment (rimbendhaou) [inline project_event/i18n/fr.po]: @jananjoy delete the following lines

**Review by rimbendhaou — APPROVED** [review]

**Comment by rimbendhaou** [comment]
@jananjoy you have to resolve conflicts

File: project_event/i18n/fr.po [hunk 7]
```diff
@@ -597,7 +608,7 @@ msgstr "Heure de début"
 #. module: project_event
 #: model:ir.model.fields,field_description:project_event.field_project_project_state
 msgid "State"
-msgstr "Statut"
+msgstr "État de tâche"
```
Comment (rimbendhaou) [inline project_event/i18n/fr.po]: @jananjoy Statut

File: project_event/data/auditlog_rule.xml [hunk 8]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo noupdate="1">
-    <record id="auditlog_event_rule" model="auditlog.rule">
-        <field name="name">Event Logs</field>
-        <field name="model_id" ref="project.model_project_project"></field>
+    <record id="auditlog_project_task_rule" model="auditlog.rule">
```
Comment (rimbendhaou) [inline project_event/data/auditlog_rule.xml:8]: @jananjoy you shouldent delete the auditlog.rule of the project_project model

File: project_event/views/auditlog_log.xml [hunk 9]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
-        <record model="ir.actions.act_window" id="action_project_event_to_auditlog_log">
-            <field name="name">Event Logs</field>
+        <record model="ir.actions.act_window" id="action_project_task_auditlog_log">
```
Comment (rimbendhaou) [inline project_event/views/auditlog_log.xml:10]: you should not delete the act_window of the project_project model

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/models/project_task.py [hunk 10]
```diff
@@ -147,6 +159,7 @@ def clean_succeeding(self):
         for preceding in self.preceding_task_ids:
             preceding.succeeding_task_ids = [(2, self.id, 0)]
 
+
```
Comment (rimbendhaou) [inline project_event/models/project_task.py]: @jananjoy empty line

File: project_event/views/auditlog_log.xml [hunk 11]
```diff
@@ -1,11 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
-        <record model="ir.actions.act_window" id="action_project_event_to_auditlog_log">
-            <field name="name">Event Logs</field>
+          <record model="ir.actions.act_window" id="action_project_event_to_auditlog_log">
```
Comment (rimbendhaou) [inline project_event/views/auditlog_log.xml]: @jananjoy indentation

**Review by rimbendhaou — CHANGES_REQUESTED** [review]

File: project_event/views/auditlog_log.xml (hunk 11 above)
Comment (jananjoy) [reply project_event/views/auditlog_log.xml]: @rimbendhaou  :+1:  Done

File: project_event/models/project_task.py (hunk 10 above)
Comment (jananjoy) [reply project_event/models/project_task.py]: @rimbendhaou :+1:  Done empty line deleted

**Review by rimbendhaou — APPROVED** [review]
looks ok

**Pull request merged**